openapi.yaml

```diff
@@ -6995,6 +6995,90 @@ paths:
       - lang: CLI
         source: >
           linode-cli longview delete 789
+  /longview/plan:
+    x-linode-cli-command: longview
+    get:
+      tags:
+      - Longview
+      summary: View Longview Plan
+      description: >
+        Returns the Longview subscription plan for this user. You can send a request to the
+        [View Longview Subscription](/api/v4/longview-subscriptions-subscription-id) endpoint for details on each particular subscription plan.
+      operationId: getLongviewPlan
+      x-linode-cli-action: plan-view
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - longview:read_only
```

Comment:
If a user has add_longview permissions, we allow them to view the plan also. Do we call that out elsewhere in docs?